commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,22 @@
      }
 
     /**
+     * calculates the page number of the result.
```

Comment:
Great that you added javadoc, but the sentence should start with a capital letter 😉